recipes/default-configure.rb

```diff
@@ -28,13 +28,19 @@
 	group "root"
 	mode "0755"
 end
+template "/usr/local/bin/archive-logs.sh" do
```

Comment:
Should this replace the previous template, or does that need to be kept?

Reply:
All of the references to `archive-logs.sh` are controlled by the `configure` phase, so in theory they should all be updated at the same time as this.